[[36m2022-08-17 10:58:38,823[39m][[34msrc.testing_pipeline[39m][[32mINFO[39m] - Instantiating trainer <pytorch_lightning.Trainer>
[[36m2022-08-17 10:58:38,829[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - Using 16bit native Automatic Mixed Precision (AMP)
[[36m2022-08-17 10:58:38,831[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - GPU available: True, used: True
[[36m2022-08-17 10:58:38,831[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - TPU available: False, using: 0 TPU cores
[[36m2022-08-17 10:58:38,831[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - IPU available: False, using: 0 IPUs
[[36m2022-08-17 10:58:38,832[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - HPU available: False, using: 0 HPUs
[[36m2022-08-17 10:58:38,832[39m][[34msrc.testing_pipeline[39m][[32mINFO[39m] - Starting testing!
[[36m2022-08-17 10:58:46,437[39m][[34mpytorch_lightning.utilities.seed[39m][[32mINFO[39m] - Global seed set to 42
[[36m2022-08-17 10:58:46,439[39m][[34mpytorch_lightning.utilities.distributed[39m][[32mINFO[39m] - Initializing distributed: GLOBAL_RANK: 0, MEMBER: 1/3
[[36m2022-08-17 10:59:22,168[39m][[34mtorch.distributed.distributed_c10d[39m][[32mINFO[39m] - Added key: store_based_barrier_key:1 to store for rank: 0
[[36m2022-08-17 10:59:22,169[39m][[34mtorch.distributed.distributed_c10d[39m][[32mINFO[39m] - Rank 0: Completed store-based barrier for key:store_based_barrier_key:1 with 3 nodes.
[[36m2022-08-17 10:59:22,170[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - ----------------------------------------------------------------------------------------------------
distributed_backend=nccl
All distributed processes registered. Starting with 3 processes
----------------------------------------------------------------------------------------------------
[[36m2022-08-17 11:00:04,634[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - Restoring states from the checkpoint path at /home/compu/jh/project/colon_compare/scripts/logs/experiments/runs/ubc/classifycompare/2022-08-11_13-33-48/checkpoints/epoch_012.ckpt
[[36m2022-08-17 11:00:14,585[39m][[34mpytorch_lightning.accelerators.gpu[39m][[32mINFO[39m] - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
[[36m2022-08-17 11:00:19,842[39m][[34mpytorch_lightning.utilities.rank_zero[39m][[32mINFO[39m] - Loaded model weights from checkpoint at /home/compu/jh/project/colon_compare/scripts/logs/experiments/runs/ubc/classifycompare/2022-08-11_13-33-48/checkpoints/epoch_012.ckpt



























wandb: Network error (ReadTimeout), entering retry loop.
datamodule is ubc
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃[1m        Test metric        [22m┃[1m       DataLoader 0        [22m┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│[36m   test/cnt_correct_diff   [39m│[35m            0.0            [39m│
│[36m       test/cnt_diff       [39m│[35m            2.0            [39m│
│[36m       test/f1_macro       [39m│[35m    0.6313838958740234     [39m│
│[36m         test/loss         [39m│[35m    0.5653321146965027     [39m│
│[36m       test/new_acc        [39m│[35m    0.8326055407524109     [39m│
│[36m      test/origin_acc      [39m│[35m    0.8330907821655273     [39m│
│[36m       test/wqKappa        [39m│[35m    0.7765989303588867     [39m│
└───────────────────────────┴───────────────────────────┘
[38mTesting[39m [38m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[39m [38m43/43[39m [38m0:00:52 • 0:00:00[39m [38m0.80it/s
[?25h